pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1798,6 +1824,26 @@ func TestPreempt(t *testing.T) {
 			if diff := cmp.Diff(sets.List(patchedPodNames), sets.List(deletedPodNames)); diff != "" {
 				t.Errorf("unexpected difference in the set of patched and deleted pods: %s", diff)
 			}
+
+			// Make sure that the DisruptionTarget condition has been added to the pod status
+			for _, patchedPod := range patchedPods {
+				expectedPodCondition := &v1.PodCondition{
+					Type:    v1.DisruptionTarget,
+					Status:  v1.ConditionTrue,
+					Reason:  v1.PodReasonPreemptionByScheduler,
+					Message: fmt.Sprintf("%s: preempting to accommodate a higher priority pod", patchedPod.Spec.SchedulerName),
+				}
+
+				_, condition := apipod.GetPodCondition(&patchedPod.Status, v1.DisruptionTarget)
+				if condition == nil {
+					t.Fatalf("expected DisruptionTarget condition on pod %s", patchedPod.ObjectMeta.Name)
+				}
```

Comment:
This check is probably redundant since we use cmp.Diff below. Please check and drop if redundant indeed.

Reply:
yeah, you are right! Dropped it!